tools/ui-components/src/container/container.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { Container } from '.';
+
+describe('<Container />', () => {
+  it('remove width when the container is fluid', () => {
+    render(
+      <Container fluid={true}>Random text to test the element width</Container>
```

Comment:
It doesn't look like this tests the width, so I'd just call it 'filler text' or something else equally generic. As it stands it suggests to the reader that the length of the text is meaningful, but it isn't.

Reply:
```suggestion
      <Container fluid={true}>Learn to code for free.</Container>
```